DISCOVER/08_grants_scholarships.md

```diff
@@ -18,7 +18,8 @@ Registration vs. Travel &amp; Lodging
 - Eligibility Requirements
   - Applicants should be 
     - from a traditionally underrepresented and/or marginalized group in the technology and/or open source communities including, but not limited to: persons identifying as LGBTQ, indigenous, women, persons of color, and/or persons with disabilities
-    - and be unable to attend without some financial assistance. (language adapted from [The Linux Foundation](https://www.linuxfoundation.org/event/api-strategy-practice-conference-2017/diversity-scholarship/))
+   -   and be unable to attend without some financial assistance [^lf-source].
+   [^lf-source]: Language adapted from The Linux Foundation's API Strategy & Practice Conference 2017 diversity scholarship announcement. See press release: [Linux Foundation Press Release](https://www.linuxfoundation.org/press/press-release/the-linux-foundation-and-open-api-initiative-announce-agenda-for-apistrat-the-api-strategy-practice-conference).
```

Comment:
Please move the footnote below the list so that the list can be rendered properly.